examples/lua_REPL/Makefile

```diff
@@ -0,0 +1,53 @@
+APPLICATION = lua_repl
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_WHITELIST := frdm-k64f iotlab-m3 mulle native nrf52840dk nucleo-f411re \
+				   stm32f4discovery
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Uncomment the following lines to enable debugging features.
+#CFLAGS_OPT = -Og
```

Comment:
I want to keep it separate. Running the full test suite requires `-DLUA_DEBUG` but that doesn't mean I want to compile with `-Og` because it may take too much flash.

Reply:
It looks like my comment was not completely clear here. What I meant is that to me, the `-Og` option is not only useful here, but could be useful for debugging other examples as well. So what I would like to propose is an option along the lines of the current `DEVELHELP`, maybe called `OPTIMIZE_DEBUG` to enable `-Og` for a random example.